cmd/istioctl/main.go

```diff
@@ -131,18 +151,18 @@ var (
 			if len(varr) == 0 {
 				return errors.New("nothing to replace")
 			}
-			for _, v := range varr {
-				if err = setup(v.Type, v.Name); err != nil {
+			for _, config := range varr {
+				if err = setup(config.Type, config.Name); err != nil {
 					return err
 				}
-				if err = model.IstioConfig.ValidateConfig(&key, v.ParsedSpec); err != nil {
+				if err = model.IstioConfig.ValidateConfig(&key, config.ParsedSpec); err != nil {
```

Comment:
same here. validation should be done server side. ideally the client should be lightweight and future-compatible.